src/net/sourceforge/kolmafia/textui/Parser.java

```diff
@@ -4570,10 +4664,53 @@ private void enforceSince(String revision, final Range directiveRange) {
     }
   }
 
+  public final void error(final String msg, final String... otherInfo) {
+    this.error(this.getCurrentPosition(), msg, otherInfo);
+  }
+
+  public final void error(final Position start, final String msg, final String... otherInfo) {
```

Comment:
Do you actually use all these overloads? (other than as helper functions)

Reply:
yes, we do